### PR TITLE
Drop CentOS 6

### PR DESCRIPTION
**Pull request by jburel**
Check that Travis remain green
All references to CentOS 6 should have been removed

### CODE REVIEW

**Comment by jburel** [comment]
I will open a tmp doc PR so it is possible to see the effect of the changes in autogen

**Comment by jburel** [comment]
TMP PR to check changes in autogen https://github.com/openmicroscopy/ome-documentation/pull/1918

**Comment by mtbc** [comment]
FYI at this point wouldn't be unreasonable to also drop Debian 8 unless we keep all still in LTS.

**Comment by jburel** [comment]
@mtbc good point. 
The doc has already been migrated to https://github.com/ome/community-docs

**Comment by jburel** [comment]
@mtbc updated

**Comment by mtbc** [comment]
There's a bit about Debian 8 and CentOS 6 across `linux/README.md` and `linux/test/README.md`.

**Comment by jburel** [comment]
done

File: linux/README.md [hunk 1]
```diff
@@ -21,12 +20,12 @@ Documentation generation
 To generate a walkthrough file corresponding to a given OS, run the
 `autogenerate.sh` script, the OS is specified as a paramater e.g.
```
Comment (mtbc) [inline linux/README.md]: not this PR's fault but maybe while here fix "parameter" too

File: linux/test/README.md [hunk 2]
```diff
@@ -64,16 +64,15 @@ It is possible to install other versions using the JAVAVER parameter.
...
 
 The supported values are: 
 openjdk18 (default), openjdk18-devel, oracle18
 
 If you do not want to install Java set JAVAVER to nojava.
 
 To add a new Java version, update the following files: 
-`step01_centos_java_deps.sh`, `step01_debian8_java_deps.sh`,
-`step01_ubuntu_java_deps.sh` and update this README.md.
+`step01_centos_java_deps.sh`, `step01_ubuntu_java_deps.sh` and update this README.md.
```
Comment (mtbc) [inline linux/test/README.md]: Maybe also `debian9`?

File: linux/README.md (hunk 1 above)
Comment (jburel) [reply linux/README.md]: done

**Comment by mtbc** [comment]
Certainly looking good to me at least. :+1:

**Comment by jburel** [comment]
@mtbc thanks for review 
we can always improve if required when we had support for ubuntu 18.04

**Pull request merged**